lib/iris/tests/__init__.py

```diff
@@ -91,6 +91,13 @@
     GRIB_AVAILABLE = True
     from iris.fileformats.grib.message import GribMessage
 
+try:
+    import iris_sample_data
+except ImportError:
+    SAMPLE_DATA_AVAILABLE = False
+else:
+    SAMPLE_DATA_AVAILABLE = True
```

Comment:
So slightly weirdly you just removed some very similar code from `iris.__init__` :stuck_out_tongue_winking_eye: